nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/NiFiRegistryResourceConfig.java

```diff
@@ -44,6 +48,14 @@
 
     private static final Logger logger = LoggerFactory.getLogger(NiFiRegistryResourceConfig.class);
 
+    @Autowired
+    private StandardProviderFactory standardProviderFactory;
+
+    @Bean
+    public EventService eventService() {
```

Comment:
I don't think this is required because `EventService` is already annotated with `@Service`, so it will be created as a bean by component scanning.